logging/cluster-logging-release-notes.adoc

```diff
@@ -11,8 +11,41 @@ toc::[]
 .Logging Compatibility
 The {logging-title} is provided as an installable component, with a distinct release cycle from the core {product-title}. The link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy] outlines release compatibility.
 
+[id="cluster-logging-release-notes-5-5-1"]
+== Logging 5.5.1
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6344[OpenShift Logging Bug Fix Release 5.5.1].
+
+[id="openshift-logging-5-5-1-enhancements"]
+=== Enhancements
+* This enhancement adds an Aggregated Logs tab to the Pod Details page of the OpenShift WebConsole when the RedHat OpenShift Logging Console Plugin is in use. This enhancement is only available on {product-title} 4.10 and later. (link:https://issues.redhat.com/browse/LOG-2647[LOG-2647])
+
+* This enhancement adds Google Cloud Logging as an output option for log forwarding. (link:https://issues.redhat.com/browse/LOG-1482[LOG-1482])
+//xref:cluster-logging-collector-log-forward-gcp.adoc
+
+[id="openshift-logging-5-5-1-bug-fixes"]
+=== Bug fixes
+* Before this update, the Operator did not ensure that the pod was ready, which caused the cluster to reach an inoperable state during a cluster restart. With this update, the Operator marks new pods as ready before continuing to a new pod during a restart, which resolves the issue. (link:https://issues.redhat.com/browse/LOG-2745[LOG-2745])
+
+* Before this update, fluentd would sometimes not recognize the kubernetes platform rotated the log file and would no longer read log messages. This update corrects that by setting the configuration parameter suggested by the upstream development team. (link:https://issues.redhat.com/browse/LOG-2995[LOG-2995])
+
+* Before this update, the addition of multi-line error detection caused internal routing to change and forward records to the wrong destination. With this update, the internal routing is correct. (link:https://issues.redhat.com/browse/LOG-2801[LOG-2801])
+
+* Before this update, changing the OpenShift WebConsole's refresh interval created an error when the 'Query' field was empty. With this update, changing interval is not an available option when the query field is empty. (link:https://issues.redhat.com/browse/LOG-2917[LOG-2917])
```

Comment:
- use an attribute for OpenShift
- web console is two words, no caps
- Per ISG, "Do not use 's to show the possessive form of brand names or product names."
- Use bold type for UI elements, not quotation marks
- `changing the interval` rather than `changing interval`?
- `query field` should be `*Query* field`